### PR TITLE
Add IDs to allow linking specific content

### PR DESCRIPTION
**Pull request by generalmanager**
Because our website doesn't have subpages there is currently no way to link to e.g. the download links or the donation page. With IDs, it's very easy to link to such content via https://syncthing.net#id

This can be useful for internal usage (links in the UI, documentation, forums) as well as external usage by bloggers and power users trying to give their loved ones no possible way to miss the download section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,7 +171,7 @@ <h4 id="download-header"></h4>
                 </div>
             </div>
             <div class="row">
-                <div class="col-sm-12">
+                <div  id="easy-to-use" class="col-sm-12">
```
Comment (AudriusButkevicius) [inline index.html]: sorry for ODC, but double space between the div and id ;)

Comment (generalmanager) [inline index.html]: Thanks, I fixed it. Is this (2nd commit) the right way to do this stuff? I was lazy used the online editor.

**Comment by generalmanager** [comment]
Yay, apparently that was the right way do do things :+1:

**Comment by AudriusButkevicius** [comment]
Ideally you would have squashed the commits. but you can't do that via the web ui ;)

**Comment by generalmanager** [comment]
Will do so on my first imperfect commit via git then ;-)

**Comment by Zillode** [comment]
OCD\* (sorry, couldn't let it go:))

**Pull request merged**